.github/workflows/ci.yaml

```diff
@@ -299,6 +299,16 @@ jobs:
           ${KAPP} deploy -a servicebinding-runtime -n apps --wait-timeout 5m -y \
             -f <(${KBLD} -f bundle/.imgpkg/images.yml -f bundle/config)
         echo "##[endgroup]"
+    - name: Test samples
+      run: |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+
+        echo "##[group]Deploy spring-petclinic"
+          ${KAPP} deploy -a servicebinding-sample-spring-petclinic -n apps --wait-timeout 5m -y \
+            -f samples/spring-petclinic
+        echo "##[endgroup]"
```

Comment:
Q: Does it make sense to have these in the same run pipeline as conformance tests?  If these break, I don't want them to gate running conformance tests (or vice-versa, for that matter).

Reply:
it's running in the `acceptance` job. I view conformance as one part of acceptance for the RI. If there's a change that breaks a basic e2e app, we know the change isn't going to pass conformant (I hope). If you prefer, I can move this after the conformance tests.

Reply:
I was hoping to have a little more granularity in tests here.  For now, it'll probably be fine (since we're still trying to get out of bootstrapping testing), but long-term I'd prefer to have separate pipeline stages for separate test suites.